pom.xml

```diff
@@ -224,7 +231,10 @@
                 <groupId>org.apache.maven.plugins</groupId>
                	<artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
-                    <argLine>-Xms512m -Xmx2048m -XX:MaxPermSize=768m -XX:ReservedCodeCacheSize=128m</argLine>
+                    <systemPropertyVariables>
+                        <org.jboss.logging.provider>jdk</org.jboss.logging.provider>
```

Comment:
A lot of these:

 2015-06-15 18:35:27 WARN   org.jboss.weld.event.ExtensionObserverMethodImpl checkRequiredTypeAnnotations() WELD-000411: Observer method [BackedAnnotatedMethod] public org.jboss.forge.furnace.container.cdi.impl.ContainerServiceExtension.processRemoteServiceTypes(@Observes ProcessAnnotatedType<Object>) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.

A lot means, 10's